dirs/dirs.go

```diff
@@ -216,15 +216,15 @@ func SnapHomeDirs() []string {
 // Homedir must be a comma separated list of the user defined home directories.
 // If homedirs is empty, SnapHomeDirs will be a slice of length 1 containing "/home".
 // Also generates the data directory globbing expressions for each user.
-// Ran at startup by configstate.Init to initialize snapHomeDirs, SnapDataHomeGlob and HiddenSnapDataHomeGlob.
+// Ran at startup by configstate.Init to initialize snapHomeDirs, snapDataHomeGlob and hiddenSnapDataHomeGlob.
```

Comment:
it's a bit strange to mention unexported  variables here,  better to say something like the values returned by and mention the exported functions instead